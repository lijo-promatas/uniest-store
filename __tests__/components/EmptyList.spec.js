import 'react-native';
import React from 'react';
import renderer from 'react-test-renderer';

import EmptyList from '../../src/components/EmptyList';

it('renders <EmptyList /> component correctly', () => {
  const tree = renderer.create(<EmptyList />).toJSON();
  expect(tree).toMatchSnapshot();
});
