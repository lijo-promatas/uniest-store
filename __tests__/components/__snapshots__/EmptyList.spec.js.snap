// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders <EmptyList /> component correctly 1`] = `
<View>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
  >
    List is empty
    .
  </Text>
</View>
`;
